apps/SageAccountWeb/package.json

```diff
@@ -5,48 +5,47 @@
   "private": true,
   "dependencies": {
     "@emotion/core": "^11.0.0",
-    "@emotion/react": "^11.10.6",
-    "@emotion/styled": "^11.10.6",
-    "@mui/icons-material": "^5.11.16",
-    "@mui/material": "^5.11.16",
-    "@mui/system": "^5.11.16",
-    "@mui/utils": "^5.11.13",
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "@mui/icons-material": "^5.13.7",
+    "@mui/material": "^5.13.7",
+    "@mui/system": "^5.13.7",
+    "@mui/utils": "^5.13.7",
     "lodash-es": "^4.17.21",
-    "plotly.js-basic-dist": "^2.22.0",
+    "plotly.js-basic-dist": "^2.24.3",
     "react": "18.2.0",
-    "react-bootstrap": "^1.5.2",
+    "react-bootstrap": "^1.6.7",
     "react-cookie": "4.0.0",
     "react-dom": "18.2.0",
-    "react-easy-crop": "^4.2.0",
-    "react-query": "3.39.1",
-    "react-router-dom": "5.3.3",
-    "react-spinners": "^0.11.0",
-    "react-use": "^17.3.1",
-    "sass": "^1.30.0",
+    "react-easy-crop": "^4.7.5",
+    "react-query": "3.39.3",
+    "react-router-dom": "^5.3.4",
```

Comment:
semver change: `react-router-dom`: `5.3.3 -> ^5.3.4` 

We were already using 5.3.4 in a few places. This fixes a peer dependency mismatch issue.